scripts/deploy.ts

```diff
@@ -24,6 +25,8 @@ async function main() {
     pxe = await setupSandbox();
     wallets = await getInitialTestAccountsWallets(pxe);
 
+    pxe.getL1ToL2MembershipWitness
```

Comment:
looks like this is incomplete?

Reply:
ah yea i think i started updating this file before creating a new one. will fix.